composer.json

```diff
@@ -1,49 +1,59 @@
 {
-    "name": "theofidry/psysh-bundle",
-    "type": "symfony-bundle",
+    "name": "teamdotblue/psysh-bundle",
     "description": "A port of the PHP REPL PsySH for Symfony.",
-    "keywords": ["console", "psysh", "shell", "REPL", "symfony", "interactive"],
     "license": "MIT",
-    "authors": [
-        {
-            "name": "Adrian Palmer",
-            "email": "navitronic@gmail.com"
-        },
-        {
-            "name": "Théo FIDRY",
-            "email": "theo.fidry@gmail.com",
-            "homepage": "https://github.com/theofidry"
-        }
+    "type": "symfony-bundle",
+    "keywords": [
+        "console",
+        "psysh",
+        "shell",
+        "REPL",
+        "symfony",
+        "interactive"
     ],
-
     "require": {
-        "php": ">=8.0",
-        "psy/psysh": "^0.11",
-        "symfony/error-handler": "^5.4|^6.0",
-        "symfony/expression-language": "^5.4|^6.0",
-        "symfony/framework-bundle": "^5.4|^6.0"
+        "php": ">=8.2",
+        "psy/psysh": "^0.11 || ^0.12",
+        "symfony/error-handler": "^6.4 || ^7.0",
```

Comment:
Symfony 5.4 is still maintained until November 2025 and we might need that support too, would be great if we could keep it in

Reply:
4.5.0 can still be installed  for that. For new features I suggest to only support one LTS Symfony version. It's FOSS after all.